certified-connectors/InEight/intro.md

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
@InEightDevOps The intro.md file and the images used in this file are not required to be open-sourced. Please remove these files from your PR and upload them in the "Functional Documentation" section in the submission form or activity control of your submission in ISV Studio.